uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1024.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1024 {
+
+  /**
+   * Removes the Search Menu heading
+   *
+   * @return bool
+   */
+  public function upgrade_1024() {
+
+    $this->up1024_removesSearchMenuHeading();
+
+    return TRUE;
+  }
+
+  /**
+   * Removes the Search Menu Heading in Admin Portal
+   *
+   */
+  private function up1024_removesSearchMenuHeading() {
```

Comment:
I think you should name the function `up1024_removeSearchMenuHeading` and not `...removes...` 🤷‍♂️